Xcode/FreeOrion.xcodeproj/project.pbxproj

```diff
@@ -972,6 +973,7 @@
 		825EA75E16DB9DB10002A797 /* ModeratorAction.cpp */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.cpp.cpp; path = ModeratorAction.cpp; sourceTree = "<group>"; };
 		825EA75F16DB9DB10002A797 /* ModeratorAction.h */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.h; path = ModeratorAction.h; sourceTree = "<group>"; };
 		82714D761ADABE680071A329 /* libboost_log.a */ = {isa = PBXFileReference; lastKnownFileType = archive.ar; path = libboost_log.a; sourceTree = "<group>"; };
+		82714D761ADABE680071A330 /* libboost_log_setup.a */ = {isa = PBXFileReference; lastKnownFileType = archive.ar; path = libboost_log.a; sourceTree = "<group>"; };
```

Comment:
This static library reference is broken, the `path` parameter points to `libboost_log.a` instead of `libboost_log_setup.a`. Don't know why building FO with this still works, but as soon as you make changes to the Xcode project, things get messed up.

@LGM-Doyle, I guess you edited the Xcode project with a text editor? Better ask someone on OSX to make such changes with Xcode, to avoid issues like this. Apparently things don't always break immediately, but at some later point, which can make fixing the broken stuff a lot harder.

Fixed in 00c389d96e21dcd43960e400caac7a23b940f9c3.